pkgs/tools/admin/azure-cli/python-packages.nix

```diff
@@ -297,6 +311,16 @@ let
         '';
       });
 
+      knack = super.knack.overrideAttrs(oldAttrs: rec {
```

Comment:
overridePythonAttrs

Reply:
I'm not sure what the difference is, is it relevant if I'm just changing the src?